File: lib-content-scrapers/src/main/java/com/ustadmobile/lib/contentscrapers/CK12/TestResponse.java
package com.ustadmobile.lib.contentscrapers.CK12;

import com.google.gson.annotations.SerializedName;

public class TestResponse {

    public Response response;

    public class Response{

        public Test test;
        public TestScore testScore;

        public class Test {

            @SerializedName("_id")
            public String id;

        }

        public class TestScore{

            @SerializedName("_id")
            public String id;

        }

    }

}

File: lib-content-scrapers/src/jvmMain/kotlin/com/ustadmobile/lib/staging/contentscrapers/ck12/practice/TestResponse.kt
package com.ustadmobile.lib.contentscrapers.ck12.practice

import com.google.gson.annotations.SerializedName

class TestResponse {

    var response: Response? = null

    inner class Response {

        var test: Test? = null
        var testScore: TestScore? = null

        inner class Test {

            @SerializedName("_id")
            var id: String? = null

            var updated: String? = null

        }

        inner class TestScore {

            @SerializedName("_id")
            var id: String? = null

        }

    }

}
